kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/PaintingTask.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.patterns.factory.tasks;
+
+public class PaintingTask implements Task
```

Comment:
jeśli to miał być immutable - brakuje final przy klasie

Reply:
Miał być. Efekt kopiowania klas. Poprawiłem.